libcob/move.c

```diff
@@ -1047,8 +1045,8 @@ optimized_move_display_to_edited (cob_field *f1, cob_field *f2)
 	c = 0x00;
 
 	for (p = COB_FIELD_PIC (f2); p->symbol; ++p) {
+		int n = p->times_repeated;
 		c = p->symbol;
```

Comment:
that's seldom used - we can "inline" `n = p->times_repeated` in the condition; as `cntr_sign_neg` and `cntr_sign_pos` are only used to `break` out (if there's floating -> at least two) and these are mutually exclusive, we should change that to a single `cntr_sign`, combining the condition for -+